src/osal/linux/osal.c

```diff
@@ -568,12 +569,61 @@ uint8_t os_buf_header(os_buf_t *p, int16_t header_size_increment)
    return 255;
 }
 
+/**
+ * Convert IPv4 address to string
+ * @param ip               In: IP address
+ * @param outputstring     Out: Outputstring. Should have length INET_ADDRSTRLEN.
+ */
+void ip_to_string(
```

Comment:
Since this is not a public function it should have static scope.

Reply:
Done